### ISSUE TITLE
vmagent ec2_sd_configs does not populate __meta_ec2_availability_zone_id  ?

### ISSUE THREAD

**Issue by jappaleinen** [VictoriaMetrics/VictoriaMetrics#3700]
### Describe the bug

Running a vmagent using docker image victoriametrics/vmagent:v1.86.2, configured with ec2_sd_configs for a number of accounts and regions using a corresponding configuration:
```
      - region: <region>
        role_arn: arn:aws:iam::<account>:role/<role>
        port: 9100
        filters:
          - name: instance-state-name
            values: [ running ]
```
Ec2-instances are discovered from all accounts and regions, and meta-information about the instances are successfully extracted, but availability-zone-id is empty everywhere.

I've verified that I can extract the az-info for the region by running `aws ec2 describe-availability-zones --region eu-central-1` after assuming the specified role. Ie role permissions seem to be correctly setup.

I've also verified that `http://vmagent:8429/service-discovery` seem to discover instances and extract `__meta__`-labels as expected. Only zone-id is empty.

### To Reproduce

```
  - job_name: 'node-exporter'
    metrics_path: '/metrics'
    ec2_sd_configs:
      - region: eu-central-1
        role_arn: arn:aws:iam::<account-1>:role/<role-a>
        port: 9100
        filters:
          - name: instance-state-name
            values: [ running ]
      - region: eu-north-1
        role_arn: arn:aws:iam::<account-2>:role/<role-b>
        port: 9100
        filters:
          - name: instance-state-name
            values: [ running ]
```

### Version

`vmagent-20230118-201742-tags-v1.86.2-0-g9785b3f57`

(`victoriametrics/vmagent:v1.86.2` docker image)

### Logs

Nothing obviously relevant detected in logs.

### Used command-line flags

`-promscrape.ec2SDCheckInterval=300s`

**Comment by hagen1778** [comment]
Hello @jappaleinen! Looks like the issue was in using the old API - see https://github.com/VictoriaMetrics/VictoriaMetrics/pull/3702

Hope it will be fixed soon and thank you for report!

**Comment by valyala** [comment]
The commit c7c4786f3fd883a3efa458a08ccad3065dd37d44 should fix the issue.

@jappaleinen , could you build vmagent from this commit according to [these docs](https://docs.victoriametrics.com/vmagent.html#how-to-build-from-sources) and verify whether it properly populate `__meta_ec2_availability_zone_id`? If it works as expected, then the commit will be included in the next release.

**Comment by jappaleinen** [comment]
I've verified the correction and it seems to populate __meta_ec2_availability_zone_id as expected!

Many thanks for resolving this issue so quickly!

**Comment by valyala** [comment]
@jappaleinen , thanks for the confirmation that the `__meta_ec2_availability_zone_id` is properly populated now! The bugfix has been included in [vmagent v1.87.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed then.

**Comment by valyala** [comment]
FYI, this issue has been also fixed in [v1.79.8 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.79.8).
